.github/workflows/website.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   publish-website:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
@dryabinin-actionengine @belom88 This fixes the install. Now fails on instanced rendering.